### ISSUE TITLE
`initialScreen` not honoring `forgotPassword` when user is logged in

### ISSUE THREAD

**Issue by doapp-ryanp** [auth0/lock#509]
Using cdn.auth0.com/js/lock/10.0/lock.min.js

If user is logged in and `Auth0Lock` is initialized with `initialScreen: "forgotPassword"`, when `.show()` is called login screen shows instead of forgot password screen.

The only workaround/hack to get this to work that I could find is to also set `allowLogin: false` when I want to show the forgot password screen.

Snippet with hack:

```
    ...
    type = type || 'login';
    var lock = new Auth0Lock('AAA', 'me.auth0.com', {
        initialScreen: type,
        allowForgotPassword: true,
        autofocus: true,
        theme: {
            logo: 'https://mylogo.png'
        },
        allowSignUp: false,
        closable: (type == 'forgotPassword'),
        allowLogin: (type != 'forgotPassword'),
        auth: {
            redirect: false,
            params: {
                scope: 'openid email',
                params: {sso: true} //must be true for MFA
            }
        },
        languageDictionary: {
            emailInputPlaceholder: "user@youremail.com",
            title: ''
        },
        loginAfterSignup: false,
        allowedConnections: ['Prod-Username-PW-Auth']
    });

    lock.show();
```

FWIW this hack was not required in v9 of the widget

**Comment by gnandretta** [comment]
Hi,

Thanks for reporting this. In the meantime you can use `rememberLastLogin: false` instead of `allowLogin: false`.

**Comment by doapp-ryanp** [comment]
I want it to remember last login though, so I can't use that.

My use case is single page app, change password flow.  I'd rather the flow go thru email, and change pw done completely on auth0.com.

**Comment by gnandretta** [comment]
This has been fixed on #534 and released in `v10.0.2`. The last login is only shown when "login" is the initial screen. The `initialScreen` option is meant to be used when you know the user needs to reset the password or sign up,  otherwise the regular flow should be used.

For advanced use cases please consider using auth0.js, you can use the [getSSOData](https://github.com/auth0/auth0.js#sso) to obtain the information used in the last login scree.

**Comment by doapp-ryanp** [comment]
Thanks for the quick turnaround on this!
